src/config.ts

```diff
@@ -9,6 +9,13 @@ const config: Config = {
       extensions: ["hardhat", "foundry", null],
       default: "hardhat",
     }),
+    typedQuestion({
+      type: "multi-select",
+      name: "Extension",
+      message: "What extensions do you want to use?",
```

Comment:
Not sure if "Which" is better than "What".

Also instead of "to use", maybe "to add" / "to install"?

(thinking out old)

Reply:
Yup makes sense !! Update at https://github.com/scaffold-eth/create-eth/pull/25/commits/a43e973275f8009ca7e682250b655303b11f24b8

used "Which extensions do you want to install?" (lol even "to add" looks good but chatGPT went with "to install")